File: api/Dtos/ServerDTOs/ServerReturnsCreatedRoom.cs
using Infastructure.DataModels;
using lib;

namespace socketAPIFirst.Dtos;

public class ServerReturnsCreatedRoom : BaseDto
{
    public int roomId { get; set; }
    public string name { get; set; }
    public int creatorId { get; set; }
    public IEnumerable<DeviceModel> deviceList { get; set; }
}

File: Frontend/src/app/Models/ServerReturnsCreatedRoom.ts
import {BaseDto} from "./baseDto";
import {DeviceModel} from "./objects/DeviceModel";

export class ServerReturnsCreatedRoom extends BaseDto<ServerReturnsCreatedRoom> {
  roomId?: number;
  name?: string;
  creatorId?: number;
  deviceList?: Array<DeviceModel>;
}
